### ISSUE TITLE
Treemacs + all-the-icons does not take effect

### ISSUE THREAD

**Issue by ZzAntares** [hlissner/emacs-doom-themes#308]
Hello, first let me say I'm very pleased with the whole doom-themes, they look awesome!, however just after I installed `treemacs` I went and enable the `(doom-themes-treemacs-config)`, however it seems the customization is not taking place, this is how I'm using the themes:

```
(use-package doom-themes
  :demand t
  :config
  (setq doom-themes-enable-bold t
        doom-themes-enable-italic t)
  (load-theme 'doom-challenger-deep)
  (doom-themes-treemacs-config)
  (doom-themes-visual-bell-config)
  (doom-themes-org-config))
```

And this is how it looks (note the screenshot shows how both `all-the-icons` and `treemacs` are configured):

![Screen Shot 2019-07-21 at 16 25 45](https://user-images.githubusercontent.com/3653680/61597301-6c4e5480-abd4-11e9-8980-6de9f690aaee.png)

Is there anything I'm missing? What I would expect to see is treemacs use all-the-icons.

**Comment by hlissner** [comment]
That is odd. This appears to be an autoloads issue. Those doom-themes-*-config functions exploit the autoloads system to load their associated extension.

As a workaround, it'd be simpler to `require` their packages, rather than use those functions:


```diff
 (use-package doom-themes
   :demand t
   :config
   (setq doom-themes-enable-bold t
         doom-themes-enable-italic t)
   (load-theme 'doom-challenger-deep)
-  (doom-themes-treemacs-config)
+  (require 'doom-themes-ext-treemacs-config)
   (doom-themes-visual-bell-config)
-  (doom-themes-org-config))
+  (require 'doom-themes-ext-org))
```

Note: `doom-themes-visual-bell-config` is the only one that actually does something, so you can keep that the way it is.

**Comment by ZzAntares** [comment]
Thank you it is working now, at first I didn't know if it was either that I did not have an up to date version of doom-themes or the suggestions you made, I think it was the former because reverting those changes keep treemacs appearance working and perhaps my outdated version was not including the fix you did in 06a6292c4e.

Thank you for the very quick reply!
